### PR TITLE
Multiple qubits and entanglement content

### LINKED ISSUE #2556: Learn content: Multiple Qubits and Entanglement
Add the card, overview page, and assets for the Multiple Qubits and Entanglement section.

### PR DESCRIPTION
**Pull request by techtolentino**
## Changes
Closes #2556 

## Implementation details

Added content card, assets, and course overview page content

Screen Shot 2022-05-10 at 9 26 57 AM

Screen Shot 2022-05-10 at 9 27 03 AM

### CODE REVIEW

File: pages/textbook-beta/course/multiple-qubits-and-entanglement.vue [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+        location: 'external-recommended-readings'
+      }
+    },
+    {
+      url: 'https://en.wikipedia.org/wiki/Quantum_Computation_and_Quantum_Information',
+      label: '(Chapters 1 and 2)',
+      segment: {
+        cta: 'quantum-computing-lecture-notes',
+        location: 'external-recommended-readings'
+      }
```
Comment (vabarbosa) [inline pages/textbook-beta/course/multiple-qubits-and-entanglement.vue:61]: all these recommended reading point to the same book/link. is this intentional?

**Review by vabarbosa — CHANGES_REQUESTED** [review]
@techtolentino nice! just had one point of clarification.

File: pages/textbook-beta/course/multiple-qubits-and-entanglement.vue (hunk 1 above)
Comment (techtolentino) [reply pages/textbook-beta/course/multiple-qubits-and-entanglement.vue:61]: good catch! I don't think these are intended to be the same... 

will take another look and fix

Comment (techtolentino) [reply pages/textbook-beta/course/multiple-qubits-and-entanglement.vue]: @vabarbosa - updated! 

BTW, as you know, we have a sync tomorrow morning to review the `ExternalRecommendedReadings` links and how we're going to structure the references... 

We can handle those updates in a separate PR, however they should change...

Comment (vabarbosa) [reply pages/textbook-beta/course/multiple-qubits-and-entanglement.vue]: yeah, i agree that can addressed in a separate PR

**Review by vabarbosa — APPROVED** [review]
mind taking care of the merge conflicts here 😁 then feel free to merge

**Pull request merged**